egs/VAD/eval.py

```diff
@@ -0,0 +1,117 @@
+import os
+import random
+
+import numpy as np
+import matplotlib.pyplot as plt
+import torch
+import json
+import argparse
+import pandas as pd
+from tqdm import tqdm
+from pprint import pprint
+from asteroid.metrics import F1_Tracker
+from binarize import Binarize
+
+from asteroid.models.conv_tasnet import VAD_Net
+from asteroid.data.vad_dataset import VADDataset
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "--md_path", type=str, required=True, help="Test directory including the csv files"
+)
+parser.add_argument(
+    "--out_dir",
+    type=str,
+    required=True,
+    help="Directory in exp_dir where the eval results will be stored",
+)
+parser.add_argument(
+    "--exp_dir",
+    type=str,
+    required=True,
+    help="Directory of the exp",
+)
+
+parser.add_argument(
+    "--n_save_ex", type=int, default=10, help="Number of audio examples to save, -1 means all"
+)
+
+parser.add_argument("--threshold", type=float, default=0.5)
+compute_metrics = ["accuracy", "precision", "recall", "f1_score"]
+
+
+class NumpyEncoder(json.JSONEncoder):
```

Comment:
Maybe it's easier to convert the stuff that is being dumped to Python types instead of using a custom encoder.